chains/evm/context/ccip-solidity-style.md

```diff
@@ -0,0 +1,422 @@
+# Structure
+
+This guide is split into two sections: [Guidelines](#guidelines) and [Rules](#rules).
```

Comment:
Do we need to copy the style guide that already exists here? Would be better to have a single source of truth as these will go out of sync